.circleci/config.yml

```diff
@@ -206,6 +234,12 @@ workflows:
             - hmpps-incentives-ui-preprod
           requires:
             - request-preprod-approval
+      - upload_test_data_to_s3:
+          context:
+            - hmpps-common-vars
+            - hmpps-incentives-ui-preprod
```

Comment:
overrides default environment (which points to `dev` namespace) with the `preprod` one.

`prod` is never accessed, but that bucket is read-only and associated k8s secret doesn't provide credentials anyway.